content/applications/inventory_and_mrp/inventory/routes/concepts/stock_warehouses.rst

```diff
@@ -1,47 +1,113 @@
+
+Create and configure a virtual parent location
+==============================================
+
+Before creating any virtual stock *Locations*, a new *Warehouse* will need to be created. This new
+Warehouse will act as a "Virtual Warehouse," and will be the "Parent" Location of other physical
+Warehouses.
 
-.. image:: stock_warehouses/ware1.png
+Create a new warehouse
+----------------------
+
+To create a new Warehouse, go to :menuselection:`Inventory --> Configuration --> Warehouses`, and
+click :guilabel:`Create`. From here, the :guilabel:`Warehouse` name can be changed, and other
+details about how to resupply the :guilabel:`Warehouse` can be configured, as well. *Routes* can
+also be set by clicking on the :guilabel:`Routes` smart button. Once the :guilabel:`Warehouse` is
+configured, virtual :guilabel:`Locations` can be created.
+
+.. image:: stock_warehouses/stock-warehouses-create-warehouse.png
    :align: center
+   :alt: The edit screen for creating a new warehouse.
 
-.. note::
-   The *Storage Location* feature will be automatically enabled. Good news, because you will need it
-   later in the process.
+Create a virtual parent location
+--------------------------------
 
-Now, you have to make sure that the main stock locations of warehouse A and warehouse B are children
-locations of the main stock location of warehouse A + B. Go to the locations menu, and edit the main
-location of your two warehouses. Then, change their parent location to main location of warehouse
-A+B.
+.. important::
+   In order to take stock from multiple Warehouses to fulfill a sales order, there need to be at
+   least **two** warehouses acting as "Child Locations" of the "Virtual Parent" Location Warehouse.
 
-.. image:: stock_warehouses/ware2.png
+To create and edit Locations, go to :menuselection:`Inventory --> Configuration --> Locations`. All
+:guilabel:`Locations` are listed here, including the stock Location of the virtual Warehouse that
+was created. Click into the stock :guilabel:`Location` for the virtual Warehouse that was
+previously created. Then, under the :guilabel:`Additional Information` tab, change the
+:guilabel:`Location Type` from :guilabel:`Internal Location` to :guilabel:`View`.
+
+This identifies this Location as a "Virtual" one, used to create a hierarchical structure for
+a Warehouse, aggregating its "Child" Locations.
+
+.. note::
+   Products can *not* be stored in this :guilabel:`Location Type`.
+
+.. image:: stock_warehouses/stock-warehouses-location-types.png
    :align: center
+   :alt: Warehouse location types in location creation screen.
+
+Configure physical warehouse locations
+--------------------------------------
 
-Sell a product from the virtual warehouse
-=========================================
+Navigate back to the :guilabel:`Locations` overview (via the breadcrumbs), then click into the
+first physical :guilabel:`Warehouse` stock :guilabel:`Location`.
```

Comment:
Need to clarify this a bit better and specify which warehouse. I think you're referring to the one you previously created. I have to clear search out to find the locations.

Reply:
You're right, good call. I added some clarification here.